### PR TITLE
Add network dashboard

### PR DESCRIPTION
**Pull request by keithf4**
# Description  

Include a network traffic dashboard in Grafana. 

Would appreciate someone that knows a bit more about what's important to monitoring networks to look over this as well as the network metrics that are provided by node_exporter, to see if it covers the important things

network_dashboard1

network_dashboard2

Fixes (issue) #169 

## Type of change  
Please check all options that are relevant  
- [ ] Bug fix (change which fixes an issue)  
- [x] Feature (change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update only  

# How Has This Been Tested?  

**Tested Configuration**:  
- [x] CentOS, Specify version(s):  7
- [x] PostgreQL, Specify version(s):  11
- [ ] docs tested with hugo <0.60  

Tested with playbook(s):  
# Checklist:  
- I have made corresponding changes to:  
    - [ ] the documentation  
    - [ ] the release notes  
    - [ ] the upgrade doc

### CODE REVIEW

**Review by hunleyd — DISMISSED** [review]

File: grafana/Network_Details.json [hunk 1]
```diff
@@ -0,0 +1,1025 @@
+{
```
Comment (sharmay) [inline grafana/Network_Details.json:1]: Possible to attach screenshot ?

**Review by sharmay — CHANGES_REQUESTED** [review]

File: grafana/Network_Details.json (hunk 1 above)
Comment (keithf4) [reply grafana/Network_Details.json:1]: Added screenshots

**Review by pgguru — DISMISSED** [review]
I can't speak to the specifics of what are being monitored, but the screenshot looks good and the idea is sound.

**Comment by hunleyd** [comment]
whats the diff bewtween 'network traffic' and 'network utilization' ?

**Comment by hunleyd** [comment]
the default metrics on https://github.com/prometheus/node_exporter sound reasonable but don't all appear to be present in the imgs that were attached? 

it looks like this dashboard includes (or can easily include) everything found at https://cdn77.my-netdata.io/#menu_system_submenu_network;theme=slate but seems like we should steal heavily from https://cdn77.my-netdata.io/#menu_ip;theme=slate

**Comment by keithf4** [comment]
> whats the diff bewtween 'network traffic' and 'network utilization' ?

Network traffic is based on an irate() of the data using a 3 minute time window to find the difference in the last two values - https://prometheus.io/docs/prometheus/latest/querying/functions/#irate

Traffic utilization is based on an increase() of the data during the last 1 minute - https://prometheus.io/docs/prometheus/latest/querying/functions/#increase

Not sure if this is accurately showing what's intended, hence my request for some assistance on deciding what is useful in this dashboard

**Review by pgguru — APPROVED** [review]

**Review by sharmay — APPROVED** [review]

**Pull request merged**